tests/e2e/check_liveness.ml

```diff
@@ -0,0 +1,57 @@
+open Feather
+open Cmdliner
+
+let curl = process "curl"
+
+let jq = process "jq"
+
+let current_main_chain_height ~rpc_node ~consensus_contract_address =
+  let url =
+    Format.sprintf "%s/chains/main/blocks/head/context/contracts/%s/storage"
+      rpc_node consensus_contract_address in
+  curl ["--silent"; url]
+  (* |. jq [".args[0].args.[0].args[2].bytes.args[0].args[0].args[0].args[1].int"] *)
+  |. jq [".args[0].args[0].args[0].args[1].int"; "-r"]
```

Comment:
We could probably use `Yojson` for a more readable parse here since we're piping the output to OCaml in the next step anyway